pkg/sysconfwatch/sysconfwatch_linux.go

```diff
@@ -92,6 +104,17 @@ func (c *SysConfWatchController) Run(ctx context.Context, ready chan<- struct{},
 	// Take a snapshot of the system and monototic clocks as a base reference
 	stimeRef, mtimeRef := getSysMonTimes()
 
+	for i := range c.iptClients {
+		iptClient := c.iptClients[i]
+		go iptClient.Monitor(
+			utiliptables.Chain("MICROSHIFT-SYSCONF-CANARY"),
+			[]utiliptables.Table{utiliptables.TableMangle, utiliptables.TableNAT, utiliptables.TableFilter},
+			func() { klog.Warningf("iptables flush is detected, restarting MicroShift"); os.Exit(0) },
```

Comment:
Need to update [this document](https://github.com/openshift/microshift/blob/main/docs/howto_sysconf_watch.md) with the new restart case.

Reply:
Updated.